training-net/src/Repositories/Interfaces/IMoviesRepository.cs

```diff
@@ -0,0 +1,9 @@
+using MvcMovie.Models;
+using Repositories.Interfaces;
+
+namespace Repositories.Interfaces
+{
+    public interface IMoviesRepository : IRepository<Movie>
```

Comment:
y convention, we use the name of the repositories like the model name, without 's': "IMovieRepository"